packages/modules/manage-versions/src/components/ManageVersions/VersionsTab/VersionsTab.tsx

```diff
@@ -42,20 +43,42 @@ const VersionsTab = (props: VersionsTabProps) => {
             id: "NAME",
             Header: stringsOverrides.name,
             accessor: "name",
+            Cell: (props: CellProps<VersionTableData | Changeset>) => {
+              return (
+                <Text>
+                  {"version" in props.row.original
```

Comment:
have simple type guard defined above component, not to repeat same code everywhere..

Reply:
Added